DirectProgramming/DPC++FPGA/Tutorials/Features/loop_ivdep/src/CMakeLists.txt

```diff
@@ -22,6 +23,9 @@ endif()
 #    For this reason, FPGA backend flags must be passed as link flags in CMake.
 set(EMULATOR_COMPILE_FLAGS "-Wall -fintelfpga -DFPGA_EMULATOR")
 set(EMULATOR_LINK_FLAGS "-fintelfpga")
+set(SIMULATOR_COMPILE_FLAGS "-Wall -fintelfpga -Xssimulation -DFPGA_SIMULATOR ${USER_SIMULATOR_FLAGS}")
```

Comment:
Please pull the latest version of the code samples, you'll see the compile and link flags have changed slightly to add `-fsycl`. So you'll have to add this to your flags as well (in this CMakeFile and the others)

Reply:
Ajay thinks it is better to use USER_HARDWARE_FLAGS for both simulation and HW, so please make this change as well

Reply:
updated.